index.html

```diff
@@ -314,8 +314,8 @@ <h2>The <dfn>PerformanceMark</dfn> Interface</h2>
         <h3>The <dfn><a>PerformanceMark</a> Constructor</dfn></h3>
         <p>The <a>PerformanceMark constructor</a> must run the following steps:</p>
         <ol>
-          <li>If the <a data-cite="HTML/webappapis.html#global-object">global object</a> is a <code>Window</code> object and <var>markName</var> uses the same name as a <a data-cite="WEBIDL#dfn-read-only">read only attribute</a> in the <code><a data-cite="NAVIGATION-TIMING#performancetiming">PerformanceTiming</a></code> interface, <a data-cite="WEBIDL#dfn-throw">throw</a> a <a data-cite="WEBIDL#syntaxerror"><code>SyntaxError</code></a>.</li>
-          <li>Create a new <a>PerformanceMark</a> object (<var>entry</var>).</li>
+          <li>If the <a data-cite="HTML/webappapis.html#current-global-object">current global object</a> is a <code>Window</code> object and <var>markName</var> uses the same name as a <a data-cite="WEBIDL#dfn-read-only">read only attribute</a> in the <code><a data-cite="NAVIGATION-TIMING#performancetiming">PerformanceTiming</a></code> interface, <a data-cite="WEBIDL#dfn-throw">throw</a> a <a data-cite="WEBIDL#syntaxerror"><code>SyntaxError</code></a>.</li>
+          <li>Create a new <a>PerformanceMark</a> object (<var>entry</var>) with the <a data-cite="ECMA-262#current-realm">current realm</a>.</li>

```

Comment:
Better to refer to the relm of the current global object since we just established which global object we're talking about.

Reply:
Done.